ERCS/erc-6538.md

```diff
@@ -1,18 +1,19 @@
 ---
 eip: 6538
 title: Stealth Meta-Address Registry
-description: A registry to map addresses to stealth meta-addresses
-author: Matt Solomon (@mds1), Toni Wahrstätter (@nerolation), Ben DiFrancesco (@apbendi), Vitalik Buterin (@vbuterin)
+description: A canonical contract for entities to register stealth meta-addresses directly or through a third party using signatures.
+author: Matt Solomon (@mds1), Toni Wahrstätter (@nerolation), Ben DiFrancesco (@apbendi), Vitalik Buterin (@vbuterin), Gary Ghayrat (@garyghayrat)
 discussions-to: https://ethereum-magicians.org/t/stealth-meta-address-registry/12888
-status: Stagnant
+status: Review
 type: Standards Track
 category: ERC
 created: 2023-01-24
+requires: 5564
```

Comment:
```suggestion
requires: 712, 1271, 5564
```

Reply:
Added in https://github.com/ethereum/ERCs/pull/259/commits/a4870dad215aff7b58518caddc0ae7e5a4d2319b